### PR TITLE
updated depreciated pytest.config at tests at pocs/tests/test_theskyx_utils.py

### PR DESCRIPTION
**Pull request by SMe12435**
updated depreciated pytest.config with request.config in line 20 which return the same config object.

### CODE REVIEW

**Comment by SMe12435** [comment]
no idea why commit affected arduinosimulator.py

**Comment by wtgee** [comment]
> no idea why commit affected arduinosimulator.py

Hi @SMe12435, the `arduinosimulator.py` is a known problem. Some of the testing is random so it returns different results each time. @jamessynge is aware of the issue.

I'll take a look at this PR in about an hour or so. Thanks!

**Comment by SMe12435** [comment]
Oh Okay!. I'd have been hunting into every code of POCS if you hadn't responded. 
Thanks a lot.

**Comment by SMe12435** [comment]
Done! @wtgee

File: pocs/tests/test_theskyx_utils.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def test_default_connect():
     If not running with a real connection then use Mocket
     """
     # Use `--with-hardware thesky` on cli to run without mock
-    if 'theskyx' not in pytest.config.getoption('--with-hardware'):
+    if 'theskyx' not in request.config.getoption('--with-hardware'):
```
Comment (wtgee) [inline pocs/tests/test_theskyx_utils.py]: You'll need to add `request` to the parameter list on line 28.

**Comment by wtgee** [comment]
Sorry, I didn't realize there was one more at:
https://github.com/panoptes/POCS/blob/develop/conftest.py#L227

It already has `request` as a param so just needs to change the one line. @SMe12435 can you do it as part of this PR so we can keep it clean? Thanks!

**Comment by SMe12435** [comment]
In order to make the change as a part of same pull request, I'll close this request and create a new one with all the changes, hope that works! @wtgee

**Pull request closed without merging**